packages/amplify-provider-awscloudformation/lib/build-resources.js

```diff
@@ -28,11 +33,14 @@ async function buildResource(context, resource) {
     !resource.lastBuildTimeStamp ||
     new Date(packageJsonMeta.mtime) > new Date(resource.lastBuildTimeStamp)
   ) {
-    const npm = /^win/.test(process.platform) ? 'npm.cmd' : 'npm';
-    require('child_process').spawnSync(npm, ['install'], { cwd: resourceDir });
+    childProcess.spawnSync(npm, ['install'], { cwd: resourceDir });
```

Comment:
A nice improvement would be to support yarn here. Bad one-liner below.
```javascript
fs.existsSync('./yarn.lock') ? (/^win/.test(process.platform) ? 'yarn.cmd' : 'yarn')  : npm
```

Reply:
I agree, that would be a nice addition to this PR. @visusnet, please add it to the list of requested changes.